src-tauri/plugins/dfu/permissions/autogenerated/reference.md

```
@@ -0,0 +1,261 @@
+## Default Permission
+
+Default permissions for the DFU plugin: enumerate DFU-capable USB devices,
+request USB permission, open/close a device, claim/release its interface,
+reset it, and perform control transfers.
+
+#### This default permission set includes the following:
```

markdownlint-cli2 (0.23.2)

[warning] Heading levels should only increment by one level at a time
Expected: h3; Actual: h4

(MD001, heading-increment)